examples/pure-hatch/src/examplePy/temporal-raw.py

```diff
@@ -0,0 +1,15 @@
+from examplePy.temperature import fahrenheit_to_celsius
+import pandas
+from typing import Sequence
+
+def calc_annual_mean(df: pandas.DataFrame):
```

Comment:
```suggestion
def calc_annual_mean(df: pd.DataFrame):
```

or just import DataFrame from pandas above?